data_model/src/query.rs

```diff
@@ -7,27 +7,36 @@ use alloc::{boxed::Box, format, string::String, vec::Vec};
 
 use derive_more::Display;
 use iroha_crypto::SignatureOf;
+use iroha_data_model_derive::{ffi_type, model};
 use iroha_macro::FromVariant;
 use iroha_schema::IntoSchema;
 use iroha_version::prelude::*;
 use parity_scale_codec::{Decode, Encode};
 use serde::{Deserialize, Serialize};
 
+pub use self::model::*;
 use self::{
     account::*, asset::*, block::*, domain::*, peer::*, permission::*, role::*, transaction::*,
     trigger::*,
 };
-use crate::{account::Account, model, Identifiable, Value};
-
-macro_rules! query {
-    ($($meta:meta)* $item:item) => {
-        crate::model! {
-            #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
-            #[derive(parity_scale_codec::Decode, parity_scale_codec::Encode)]
-            #[derive(serde::Deserialize, serde::Serialize)]
-            #[derive(iroha_schema::IntoSchema)]
-            $($meta)*
-            $item
+use crate::{account::Account, Identifiable, Value};
+
+macro_rules! queries {
+    ($($($meta:meta)* $item:item)+) => {
+        pub use self::model::*;
+
+        #[iroha_data_model_derive::model]
+        pub mod model{
+            use super::*;
+
+            $(
+                #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+                #[derive(parity_scale_codec::Decode, parity_scale_codec::Encode)]
+                #[derive(serde::Deserialize, serde::Serialize)]
+                #[derive(iroha_schema::IntoSchema)]
+                $($meta)*
+                $item
+            )+
         }
```

Comment:
```suggestion
        pub mod model{
            use super::*; $(
                #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
                #[derive(parity_scale_codec::Decode, parity_scale_codec::Encode)]
                #[derive(serde::Deserialize, serde::Serialize)]
                #[derive(iroha_schema::IntoSchema)]
                $($meta)*
                $item )+
        }
```

totally irrelevant, but this is how I prefer for macro to look like because it's the closest to how the actual source code would be laid out so it makes it easier for me to read it

Reply:
I format the code as if it is source code and then put macro tokens around it